### PR TITLE
Fix istioctl install command for OpenShift

### PR DESCRIPTION
**Pull request by sridhargaddam**
Fixes: https://github.com/istio/istio.io/issues/16047

### CODE REVIEW

**Review by kfaseela — APPROVED** [review]

**Review by bleggett — COMMENTED** [review]
this isn't correct, `profile` and `platform` are not the same.

You can for instance have something like `--profile=ambient --platform=openshift`

**Comment by bleggett** [comment]
@sridhargaddam can you correct? `platform` and `profile` are 2 different things, and you can have both.

`openshift` is a platform, not a profile.

File: content/en/docs/setup/platform-setup/openshift/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Follow these instructions to prepare an OpenShift cluster for Istio.
 Install Istio using the OpenShift profile:
 
 {{< text bash >}}
-$ istioctl install --set global.platform=openshift
+$ istioctl install --set profile=openshift
```
Comment (bleggett) [inline content/en/docs/setup/platform-setup/openshift/index.md:19]: This should be `istioctl install --set values.global.platform=openshift` - using the Helm values.

istioctl-only profiles via `--set profile` are/should be deprecated, and `openshift` is now a `platform`, and not a `profile`.

This is so we can compose like so: `istioctl install --set values.global.platform=openshift  --set values.global.profile=ambient`

**Comment by bleggett** [comment]
NVM, fixed in https://github.com/istio/istio.io/pull/16051

File: content/en/docs/setup/platform-setup/openshift/index.md (hunk 1 above)
Comment (sridhargaddam) [reply content/en/docs/setup/platform-setup/openshift/index.md:19]: Okay Ben. We can continue with your PR - https://github.com/istio/istio.io/pull/16051/files

**Pull request merged**